### PR TITLE
PSW-33: npm updates

### PR DESCRIPTION
**Pull request by WunderNieves**
https://wunder.atlassian.net/browse/PSW-33

Changes proposed in this PR:
- Update npm packages
- Solve vulnerabilities

How to test:
1. Run "npm install" to get the packages updated
2. Run "npm run lint" to make sure that linting is working properly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,22 +8,22 @@
...
-    "@typescript-eslint/parser": "^4.28.0",
-    "eslint": "^7.27.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-prettier": "^3.4.0",
-    "eslint-plugin-react": "^7.23.2",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "prettier": "^2.3.2",
-    "typescript": "^4.3.2"
+    "@types/node": "^17.0.23",
```
Comment (badrange) [inline package.json:15]: I think it is better to use types/node 16.x if the project uses node 16 (latest LTS version of node)

File: package.json [hunk 2]
```diff
@@ -8,22 +8,22 @@
...
+    "@types/react": "^17.0.43",
+    "@typescript-eslint/eslint-plugin": "^5.18.0",
+    "@typescript-eslint/parser": "^5.18.0",
+    "eslint": "^8.12.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-react-hooks": "^4.4.0",
+    "prettier": "^2.6.2",
```
Comment (lauriskuznecovs) [inline package.json:25]: After running `npm run lint` there was an deprecation warning:
```
jsxBracketSameLine is deprecated.
```

According to https://github.com/SquareFactory/prettier-config/issues/1 this options has been deprecated from Prettier 2.4
Lets remove `jsxBracketSameLine` from `.prettierrc`

File: package.json (hunk 1 above)
Comment (lauriskuznecovs) [reply package.json:15]: Agree

**Review by lauriskuznecovs — CHANGES_REQUESTED** [review]
@WunderNieves There are 2 items to update, see comments.

**Review by lauriskuznecovs — APPROVED** [review]
Production build and linting works fine.

**Pull request merged**